### PR TITLE
Copy and style changes

### PR DESCRIPTION
**Pull request by techbelly**

### CODE REVIEW

**Comment by DougC** [comment]
I've created a review app for this PR on Heroku at https://beis-ppr-dev-pr-50.herokuapp.com which you can use to check the changes look right.

File: src/main/twirl/views/questionnaire/start.scala.html [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+        <li>what information you need to gather before you file your report</li>
     </ul>
 
-    <p class="panel">
-        You will now be asked some questions about your business.
-        <span class="bold-small">Your answers must be for your business on its own, even if it's part of a group.
-            Every business within the group will need to do this individually.</span>
+    <p>
+        Your answers must be for your business on its own, even if it's part of a group.
+        Every business within the group will need to do this individually.
```
Comment (dgmstuart) [inline src/main/twirl/views/questionnaire/start.scala.html]: The copy change is correct, but they've done usability testing which says that people missed this block

Solution: apply the previous formatting applied to this block: panel and bold

File: src/main/twirl/views/shared/_coverageInformation.scala.html [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
+    <li>It is for goods, services or intangible property, including intellectual property</li>
 
-    <li>It is not for <a rel="external" target="_blank" href="http://www.legislation.gov.uk/ukpga/2015/26/section/2/enacted">financial services</a>.</li>
+    <li>It is not for <a rel="external" target="_blank" href="http://www.legislation.gov.uk/ukpga/2015/26/section/2/enacted">financial services</a></li>
 </ul>
 
 <h3 class="heading-small">Significant Connection to the UK</h3>
 
 <p>
-    If a UK law (i.e. the law of England and Wales, Scotland or Northern Ireland) applies to a contract without the parties
```
Comment (dgmstuart) [inline src/main/twirl/views/shared/_coverageInformation.scala.html]: Everything from here below will take longer to sign off and therefore should be in a separate PR

File: src/main/twirl/views/shared/_day1Calculation.scala.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-    whether by post, email or otherwise. This means that you need to calculate from the day after receipt (i.e. the
-    day after receipt is day 1). The end of the period is when payment is made to the supplier. A payment is
-    counted as 'made' when it is available to the supplier, (for example, when it clears their bank account) unless
-    delayed by circumstances out of your control. You do not need proof that your payment has cleared, but you need to
-    count the standard time it takes to clear if there are no unforeseen issues.
-</p>
\ No newline at end of file
+<strong class="bold-medium">Calculating the number of days taken to pay suppliers</strong><br />
+When you work out how many days you take to pay suppliers, day 0 is the day you receive their invoice or timesheet, whether by post, email or otherwise.<br />
+Day 1 for calculation purposes is the day after receipt - whatever day of the week that is, including weekends and bank holidays. <br />
```
Comment (dgmstuart) [inline src/main/twirl/views/shared/_day1Calculation.scala.html]: Remove "and bank holidays"

**Review by dgmstuart — CHANGES_REQUESTED** [review]
Alex Jackson communicated some changes that need to be reverted here. I'll progress them.

**Comment by dgmstuart** [comment]
I pulled down the latest changes from master onto our fork and rebased this PR on top of that

**Comment by DougC** [comment]
@dgmstuart I've invited you as a collaborator on this repo so you can work directly and avoid all the hassle with updating your fork.

**Comment by dgmstuart** [comment]
@DougC ah thanks - hopefully we're good now, but if further changes need to be made then we can re-create these changes on the main project and raise a new PR directly (instead of from the fork).

**Review by DougC — APPROVED** [review]
Corinne has confirmed she's happy for these changes to be merged

**Comment by dgmstuart** [comment]
Great - thanks for merging Doug. I figure if those other changes do get approved later then you can just do a git revert on the commits where I removed them. 

Hope that makes sense!

Duncan Stuart

On Thu, 31 Aug 2017 at 12:47 Doug Clinton

<
mailto:Doug Clinton <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

@DougC

approved this pull request.

Corinne has confirmed she's happy for these changes to be merged

—

You are receiving this because you were mentioned.

Reply to this email directly,
https://github.com/UKGovernmentBEIS/payment-practices-reporting/pull/50#pullrequestreview-59832474
, or

**Comment by DougC** [comment]
@dgmstuart I'd rather not get involved in that, to be honest  :-) I'm not sure which changes you mean, but best if you pull them onto another PR as and when they're needed.

**Pull request merged**